src/program/lwaftr/tests/config/selftest.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
-set -o errexit
```

Comment:
Exit codes are now checked explicitly for better feedback, and tests do not stop anymore upon the first error.